lib/ReactViews/ExplorerWindow/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,238 @@
+import React, { FC, useEffect, useState } from "react";
+
+import { useTranslation } from "react-i18next";
+import { runInAction } from "mobx";
+import { observer } from "mobx-react";
+import styled, { useTheme } from "styled-components";
+
+import isDefined from "../../../Core/isDefined";
+
+import CatalogMemberMixin from "../../../ModelMixins/CatalogMemberMixin";
+import GroupMixin from "../../../ModelMixins/GroupMixin";
+import MappableMixin from "../../../ModelMixins/MappableMixin";
+
+import { BaseModel } from "../../../Models/Definition/Model";
+
+import ViewState from "../../../ReactViewModels/ViewState";
+
+import Box from "../../../Styled/Box";
+import { RawButton } from "../../../Styled/Button";
+import Ul, { Li } from "../../../Styled/List";
+import { TextSpan } from "../../../Styled/Text";
+
+import { DataCatalogTab } from "./DataCatalogTab";
+import { MyDataTab } from "./MyDataTab";
+
+interface ITabsProps {
+  viewState: ViewState;
+  onClose: () => void;
+}
+
+export const Tabs: FC<ITabsProps> = observer(({ viewState, onClose }) => {
+  const { t } = useTranslation();
+  const theme = useTheme();
+
+  const [tabs, setTabs] = useState<any[]>([]);
+
+  const { terria } = viewState;
+
+  useEffect(() => {
+    const myDataTab = {
+      title: "my-data",
+      name: t("addData.myData"),
+      category: "my-data",
+      panel: (
+        <MyDataTab
+          viewState={viewState}
+          onFileAddFinished={onFileAddFinished}
+        />
+      )
+    };
+
+    if (terria.configParameters.tabbedCatalog) {
+      setTabs(
+        ([] as any[]).concat(
+          terria.catalog.group.memberModels
+            .filter(
+              member =>
+                member !== terria.catalog.userAddedDataGroup &&
+                CatalogMemberMixin.isMixedInto(member)
+            )
+            .map((member, i) => ({
+              //@ts-ignore
```

Comment:
I think `//@ts-ignore` could be removed by changing line 57 to:
```ts
(member): member is CatalogMemberMixin.Instance
```
and maybe using `member.nameInCatalog!` if name properties appear as `string | undefined`

Reply:
done